packages/ui/src/components/card.tsx

```diff
@@ -23,22 +23,25 @@ export const Card = withVariants<
 >(
   {
     $defaults: {
-      backgroundColor: '$cardBackgroundColor',
-      borderRadius: '$card',
+      backgroundColor: '$white',
+      borderRadius: 28,
       borderWidth: 0,
       cursor: 'pointer',
       flexDirection: 'column',
-      padding: '$card',
+      margin: 9,
+      overflow: 'hidden',
+      padding: 36,
+      marginVertical: 40,
+      marginHorizontal: 20,
+      gap: 10,
```

Comment:
Why did you change the style of the `Card` component? I think it was correctly style.